package.json

```diff
@@ -5,44 +5,38 @@
   "engine": "node 6.11.5",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint --fix .",
-    "react-dev": "webpack -d --watch",
-    "server-dev": "nodemon server/index.js",
+    "festart": "webpack -d --watch",
+    "bestart": "nodemon --inspect server/index.js",
```

Comment:
What is the reason for the change in the name?

Reply:
its a slightly easier convention to remember, imo.